using_images/other_images/jenkins.adoc

```diff
@@ -117,6 +117,87 @@ define a persistent volume pool in the OpenShift deployment.
 
 Once selected, you must link:../../dev_guide/templates.html#creating-resources-from-a-template[instantiate] the template to be able to use Jenkins.
 
+== Customizing Jenkins image
+
+=== Using Jenkins image as Source-To-Image builder
+
+The Source-To-Image allows you to do additional modifications to the Jenkins
```

Comment:
tool ? i thought we're talking about the "strategy" :-) tools sounds to me like CLI